### PR TITLE
[wip] Unpin jedi

### PR DESCRIPTION
**Pull request by bollwyvl**
ipython 7.20 just landed (as did seleniumlibrary 5), let's see if everything's cool!

[Binder](https://mybinder.org/v2/gh/bollwyvl/jupyterlab-lsp/unpin-jedi?urlpath=lab%2Ftree%2Fexamples%2FPython.ipynb)

## References

- reverts pin from #445

## Code changes

n/a

## User-facing changes

- removes `jupyter-offlinenotebook` from binder so it doesn't pop the build warning

## Backwards-incompatible changes

n/a

## Chores

- [ ] linted
- [ ] tested
- [ ] documented
- [ ] changelog entry

### CODE REVIEW

**Comment by bollwyvl** [comment]
Graah, as with https://github.com/jupyter/jupyter_client/pull/615, we want to unpin jed, _but only on py3.7+_...

File: binder/postBuild [hunk 1]
```diff
@@ -18,6 +18,9 @@ python -m pip install -e . --ignore-installed --no-deps -vv
 jupyter labextension develop . --overwrite
 popd
 
+# Clean up incompatible extension
+jupyter labextension uninstall --no-build jupyter-offlinenotebook || echo "ok"
+
```
Comment (krassowski) [inline binder/postBuild:23]:
```suggestion

```

No longer needed as per https://github.com/jupyterhub/repo2docker/pull/996

**Comment by bollwyvl** [comment]
Rad! While a useful feature, I still dislike how much space it takes up in
the toolbar, but it's one less thing to do.

**Comment by krassowski** [comment]
So we have a `pip check` failing. Unpinning `jedi` solves part of the issue (with pyls), but there is still:

`autopep8 1.5.6 has requirement pycodestyle>=2.7.0, but you have pycodestyle 2.6.0.`

**Comment by krassowski** [comment]
Though pyflakes 2.3 seems to be on conda-forge: https://github.com/conda-forge/pyflakes-feedstock ...

**Comment by bollwyvl** [comment]
So... we're not getting very helpful feedback, but I think what we're probably seeing is getting a partial solved for a `pyls` that is so old it (wrongly) doesn't have a jedi pin. On 3.6, this is pulling in the three year old `0.11.1`:

```
python-language-server    0.11.1                   py36_0    conda-forge
```

So I don't think we can really unpin jedi until `python-ls` has a release...

**Comment by krassowski** [comment]
Merging should solve #598.

**Pull request merged**